### DISCUSSION TITLE
Propose a simple migration from classic `canvas` to this module for `pdf.js`

### DISCUSSION THREAD

**Discussion by cmahnke** [Brooooooklyn/canvas#933] [General]
It should be possible to use vanilla `pdf.js` with this library using.  After [this comment](https://github.com/mozilla/pdf.js/issues/15652#issuecomment-2459692848) I would like to propose to provide a copy and paste ready solution...
Currently I'm strugeling in setting this up:
* For `npm` something like [overrides](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides) should work
* For `yarn` there was `resolutions` - but that only worked with versions.

But (real, as in transitive) dependency management doesn't seem to be a strength of both...

**Comment by Brooooooklyn** [comment]
pdf.js now is using the `@napi-rs/canvas`
